test/e2e/network/util.go

```diff
@@ -21,9 +21,17 @@ import (
 	"fmt"
 	"time"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: import alias with the same name does not help

Reply:
Yeah, I know that.
This part comes from the original one, I don't have strong opinion about that.

Reply:
I have a chance to update this PR.
So I am applying this your point into the latest PR.